src/components/Redirect/LoaderTimer.tsx

```diff
@@ -0,0 +1,38 @@
+import Button from '@/components/Button';
+
+import RedirectIcon from '../icons/redirect';
+
+interface LoaderTimerProps {
+    timer: number;
+    goButtonClickHandler: () => void;
+}
+
+const LoaderTimer = ({ timer, goButtonClickHandler }: LoaderTimerProps) => {
+    if (timer < 1) {
+        return (
+            <div className="mt-4 flex flex-col items-center space-y-2">
+                <RedirectIcon />
+                <p className="text-1xl">Redirecting...</p>
+            </div>
+        );
+    } else {
```

Comment:
Do we need this else block?

Reply:
Both returns are in the same block so we need this else block

Reply:
You are **returning** when timer is less than 1. So if you remove the else block you will only remove a level of nesting. Functionality will remain same.

Reply:
Done